inventory/internal/store/os_test.go

```diff
@@ -159,6 +160,20 @@ func Test_Create_Get_Delete_Update_Os(t *testing.T) {
 			},
 			valid: true,
 		},
+		"CreateBadOsDuplicateMetadata": {
```

Comment:
Any test case to validate content that is not key-value map or empty string?

Reply:
Done, added 1 more here, and more in the util_test